src/_leftpad.js

```diff
@@ -0,0 +1,11 @@
+// Underscore.js-like wrapper to left pad a string to a certain length with a character
+export default function _leftpad(str, len, ch) {
```

Comment:
is this going to be used in node.js only ? , if so then use String.prototype.repeat instead